src/quo/foundations/colors.cljs

```diff
@@ -239,7 +239,7 @@
                60 "#CC6438"}
    :army      {50 "#216266"
                60 "#1A4E52"}
-   :flamingo  {50 "#F66F8F"
+   :pink      {50 "#F66F8F"
                60 "#C55972"}
```

Comment:
These renames are needed to properly handle the `:pink` color coming from desktop via status-go.

Reply:
@ulisesmac - `status-go` has `flamingo` in the constants. 

https://github.com/status-im/status-go/blob/625c11ced41e35e7893d4aecc65f03c1ee71a2f4/multiaccounts/common/const.go#L19

https://github.com/status-im/status-go/pull/4502

I guess the Desktop app hasn't changed from `:pink` to `:flamingo` or something went wrong.

Just a caution: If we change back to `pink`, this will affect the existing accounts created from mobile using the `flamingo` color.

Reply:
Hmm... 🤔

Status-go is returning the color as `:pink`, not as `:flamingo` and it exists as `:pink` on figma.

We haven't released the app officially, so I think we don't need to support accounts created with `:flamingo`. Wdyt?

I will improve this PR by adding a fallback for unknown customization colors (such as this example or the `:yin-yang` variant).

Reply:
> Status-go is returning the color as :pink, not as :flamingo and it exists as :pink on figma.

Interesting 🤔 . My guess is the account is created from Desktop and used `pink` and it's synced with mobile which tries to find `pink` ( `flamingo`) and fails.

I guess we should change it back to `pink` in `status-go` If it's pink in the Design system. Not sure why we changed to `flamingo`. Nevertheless, we should change it back.

> We haven't released the app officially, so I think we don't need to support accounts created with :flamingo. Wdyt?

Yeah, The caution is just for QA. 

> I will improve this PR by adding a fallback for unknown customization colors (such as this example or the :yin-yang variant).

Yes, for sure.  I guess we should check with the list we support and update it to fallback (blue/primary) if we don't support. In that way, we keep the app usable without any crash. WDYT?

https://github.com/status-im/status-mobile/blob/a5c81824613ca931e9a770506b1d1c4f686cd58e/src/status_im/contexts/profile/rpc.cljs#L17

IMO, we should remove the `yin-yang` from the desktop to keep it the same as mobile.

Reply:
@smohamedjavid 

> Yes, for sure. I guess we should check with the list we support and update it to fallback (blue/primary) if we don't support. In that way, we keep the app usable without any crash. WDYT?

Yes, that's what I had in mind, that is what Desktop does, if we create an account with `:flamingo` desktop uses blue/primary.

Reply:
@smohamedjavid

Instead of applying the fallback during parsing, since there'll be a lot of places where we receive a customization color and it's not only for a profile (e.g. a watched account, others' profiles, etc). I added the fallback in the foundations file, when we attempt to resolve an unknown color it fallbacks to `:blue`.

LMK if this solution fits your expectations and if you agree with it :)